app/controllers/api_controller/collection_config.rb

```diff
@@ -0,0 +1,110 @@
+class ApiController
+  class CollectionConfig < Config::Options
+    def collection?(collection_name)
+      c(collection_name)[:options].include?(:collection)
+    end
+
+    def custom_actions?(collection_name)
+      cspec = c(collection_name)
+      cspec && cspec[:options].include?(:custom_actions)
```

Comment:
This pr is simple extraction of existing code.

In all the other cases it is assured that cspec exists. In this case it is not.

I haven't had a strong preference (efficiency vs consistency) here, so I left it as it is.